src/main/res/values/strings.xml

```diff
@@ -173,6 +173,8 @@
 
     <string name="alert_delete_dialog">Delete file</string>
     <string name="alert_delete_confirm">Are you sure?</string>
+    <string name="never_opened">Never Opened</string>
+    <string name="last_opened">Last Opened</string>
```

Comment:
Please add `never_opened` and `last_opened` for `values-zh` (Chinese) too. Rest seems fine.

Reply:
Sorry for the delay. I'll do it now

Reply:
Done please check